cypress/integration/admin/pageBuilder/page/createPage.spec.js

```diff
@@ -1,77 +1,59 @@
-//         });
-//
-//         cy.findByTestId("pb-editor-back-button").click();
-//
-//         // Wait till the "/pages" route
-//         cy.findAllByTestId("new-record-button").first().should("exist");
-//
-//         cy.findByTestId("default-data-list").within(() => {
-//             cy.get(".mdc-list-item")
-//                 .first()
-//                 .within(() => {
-//                     cy.findByText(newPageTitle).should("exist");
-//                     cy.findByText(/Static/i).should("exist");
-//                     cy.findByText(/Draft/i).should("exist");
-//                     cy.findByText(/\(v2\)/i).should("exist");
-//                 });
-//         });
-//
-//         // Delete page
-//         cy.findByTestId("pb-page-details-header-delete-button").click();
-//         cy.findByRole("alertdialog").within(() => {
-//             cy.findAllByTestId("dialog-accept").next().click();
-//         });
-//
-//         cy.findByTestId("default-data-list").within(() => {
-//             cy.get(".mdc-list-item")
-//                 .first()
-//                 .within(() => {
-//                     cy.findByText(newPageTitle).should("not.exist");
-//                 });
-//         });
-//     });
-// });
+context("Page Builder Page Creation", () => {
+    beforeEach(() => {
+        cy.login();
+    });
+
+    describe("When creating new page", () => {
+        const id = uniqid();
+
+        afterEach(() => {
+            // Here we can pass id (this id is the part of the page title) instead of full page title,
+            // because query can search not only by a full title, but also by some specific symbols in our case it is an id.
+            cy.pbDeleteSpecificPage(id);
+        });
+
+        it("should be able to create page, publish page and check whether it exists in the list of the pages", () => {
+            const newPageTitle = `Test page ${id}`;
+
+            cy.visit("/page-builder/pages");
+
+            // Button is hidden so we need to force click it.
+            cy.findByTestId("new-page-button").click({ force: true });
+            // Redirects us to the Page Builder Editor page (route "/page-builder/editor/").
+            cy.findByTestId("create-blank-page-button").click();
+            // Check if we got redirected to the Page Builder Editor (route "/page-builder/editor/").
+            cy.url().should("includes", "/page-builder/editor/");
+            // Check if we are on the Page Builder Editor page (route "/page-builder/editor/") and it is empty.
+            cy.findByTestId("pb-content-add-block-button", { timeout: 15000 });
+
+            // Changing default name for the page (because default name is Undefined) so we can use it in the query to find it's id,
+            // so we can use that id to delete the page.
+            cy.findByTestId("pb-editor-page-title").click();
+            cy.get(`input[value="Untitled"]`).clear().type(newPageTitle).blur();
```

Comment:
Actually we can get to the input via `pb-editor-page-title` test ID.

Reply:
Maybe you are right, but I think it may be impossible because when we click on "pb-editor-page-title" it renders input instead of title, and that rendered input has no data-testid. Besides that if we try to add `.clear().type(newPageTitle).blur()` on `findByTestId("pb-editor-page-title")` we will get following error.
<img width="1317" alt="Знімок екрана 2023-08-11 о 18 46 56" src="https://github.com/webiny/webiny-js/assets/129154672/6c099ede-b486-4f3d-8b70-e7bd336d7026">

Maybe it would be better if we just add data-testid for that input?

Reply:
I understand, all good, won't dwell on this.